submissions/kt-std/friends-app/index.html

```diff
@@ -0,0 +1,66 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;1,700&display=swap" rel="stylesheet">
+    <title>Friends App</title>
+</head>
+
+<body>
+    <header class="header__container flex">
+        <h3 class="logo__text">Fri<span class="colored">end</span>s App</h3>
+        <div class="search__container">
+            <span class="info flex"></span>
+            <input type="text" class="search__field" id="search" placeholder="Search...">
+            <label class="before"></label>
+        </div>
+    </header>
+    <main class="main__container flex">
+        <div class="main__row flex">
+            <h2 class="heading">Your friends <span class="amount"></span></h2>
+            <button class="filters__button" id="showFiltersButton">Show filters</button>
+        </div>
+        <div class="filters__container">
+            <div class="select__item">
+                <label class="label" for="sort">Sort:</label>
+                <ul class="select__container" id="sort">
```

Comment:
@zonzujiro, I'm so grateful for your time and truly appreciate your help!  :bow:
I've replaced these nested `div` tags with the `form` and `fieldset`. As for `inputs` :eyes: First, I tried to implement the sort field with the `<select>`. But, unfortunately, it didn't succeed because styling the select drop-downs' with pure CSS seems quite impossible as the select element is inherited from the operating system, rather than provided by the browser ([found it there](https://www.sitepoint.com/select-html-element/)). Therefore, I've decided to make the custom "select" via lists and then style it for my needs. 
I've removed one `ul` and rewritten the code a bit. There left only `span` and `ul`. Please let me know if it is still needed to replace sort "select" via inputs (e.g. radio buttons) for providing better UX.

P.S. Fingers crossed-tight for this to looks better now  :sweat_smile:

Reply:
Ok, if you want to have your own select, then you should follow this guide https://www.w3.org/TR/wai-aria-practices/examples/listbox/listbox-collapsible.html

And in such case form and fieldset is unnecessary. You don't have any inputs in there.

Reply:
I've updated the select due to the given guide. Enjoy accessing it with  the keyboard  :relieved: